sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -463,6 +463,7 @@ aditof::Status CameraItof::setMode(const uint8_t &mode) {
                 return localStatus;
             }
 
+#ifdef TARGET
```

Comment:
The CameraItof object is and should continue to be indifferent of the type of communication with the depth sensor. It doesn't care if it's talking to the sensor over network, usb or directly (on target). Threfore #ifdef TARGET should not be present in CameraItof.
There are multiple ways to handle this. You could leverage the fact that control "depthComputeOpenSource" doesn't exists in OfflineDepthSensor. Or you could make the OfflineDepthSensor have a special value for this control and check that before printing. You can even use the name of the sensor to differentiate between offline and other configurations.